index.html

```diff
@@ -31,8 +31,22 @@
         }
     </script>
     <body>
+        <div id="drop-zone">
+            <svg
+                class="icon"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 10 10"
+            >
+                <path
+                    d="M5,3 l0,4 M3,5 l4,0"
+                    fill="none"
+                    stroke="#fff"
+                    stroke-width="0.5"
+                />
+            </svg>
+        </div>
         <div id="topBar">
-            <div id="drop-zone" class="topBarDiv dropZone">
+            <div class="topBarDiv dropZone">
```

Comment:
If the whole area is a drop-zone now, is this class needed? From the CSS, it looks like all it does is adjust padding.

Reply:
That does look unnecessary - removing, thanks!